cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -230,7 +230,7 @@ class ScannerCommand : OrtCommand(
 
             val ortResult = readOrtResult(input)
             return runBlocking {
-                scanner.scan(ortResult, skipExcluded)
+                scanner.scan(ortResult, skipExcluded, labels)
```

Comment:
Commit message nits:
- Make "after the scanner has run" the last part of the sentence.
- "by CLI parameter" -> "on the command line"